dotCMS/src/main/java/com/dotcms/rest/WorkflowResource.java

```diff
@@ -48,7 +49,7 @@
 	@Consumes(MediaType.APPLICATION_JSON)
 	public Response fireWorkflow(@Context HttpServletRequest request,
 			String json) throws JsonProcessingException, IOException,
-			DotContentletStateException, DotDataException, DotSecurityException {
+			DotContentletStateException, DotDataException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'com.dotmarketing.portlets.contentlet.business.DotContentletStateException' which is a runtime exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)